src/components/TextButton/TextButton.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TextButtonWrapper } from './TextButton.styles';
+
+const TextButton = ({ content }) => {
+  return (
+    <TextButtonWrapper>
```

Comment:
You can simplify this by making `<TextButtonWrapper>` the actual button element. See comment on styles.

Reply:
Can you also add "type" as a prop, then pass it it "button by default like so:

```
  const TextButton = ({ context, type = "button" }) => {
    ...
  }
```

Reply:
You'll probably also want to pass in a prop for the function whenever the button get's pressed:

```
const TextButton = ({ content, type = "button", onClick }) => {
     return (
        <TextButtonWrapper onClick={onClick} type={type}>
            {content}
        </TextButtonWrapper>
    )   
 };
```

I'd then also write a test that make sure that function correctly executes.

Reply:
I've used a Jest Mock function to test if function has executed. Not sure if that's right.